docgen/src/guide/index.md

```diff
@@ -10,11 +10,10 @@ React InstantSearch is the ultimate toolbox for creating instant search experien
 
 Algolia is a hosted Search API. React InstantSearch is built on top of it to make it easy to integrate of Algolia into your React applications.
 
-This means you need to have an account on Algolia in order to use React InstantSearch. Signup with
-your GitHub account at https://www.algolia.com/users/sign_up, there's a 14 days trial.
+You will need an Algolia account to use React InstantSearch. If you don't already have one, sign up using GitHub at https://www.algolia.com/users/sign_up. New accounts are automatically enrolled in an unlimited 14-day trial. At the end of the trial you can upgrade to a paid plan or choose the Hacker Plan, which is completely free and lets you store up to 10,000 records.
```

Comment:
Maybe it's worth noting that Github is not the only mean to create an account and we also support Google and vanilla signups. I fear that some people might be intimidated by the fact it requires strictly a github account. WDYT?